hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetContentsHandler.java

```diff
@@ -65,8 +66,8 @@ public FileGetContentsHandler() {}
     public void validate(@NonNull final QueryContext context) throws PreCheckException {
         final var query = context.query();
         final FileGetContentsQuery op = query.fileGetContentsOrThrow();
-        if (op.hasFileID()) {
-            validateFileExistence(op.fileID(), context);
```

Comment:
I think this means we can remove the `validateFileExistence` method entirely from FileQueryBase...

Reply:
Sorry I thought I removed it thanks.